tools/api-docs-generator/src/filters/TypeScriptFilters.ts

```diff
@@ -176,7 +176,11 @@ function removeDuplicateVariantTypes(releases: MinecraftRelease[]) {
                     return (
                         index ===
                         self.findIndex(value => {
-                            return value.name === item.name;
+                            return item.name === 'array'
```

Comment:
Mind splitting this out into code that isn't a single statement for readability?